openvpn3/session.go

```diff
@@ -128,7 +125,17 @@ func (session *Session) Start() {
 			session.resError = ErrInitFailed
 			return
 		}
-		session.sessionPtr = sessionPtr
+
+		go func() {
+			session.finished.Wait()
```

Comment:
You cannot reuse finished sync group to signal stop - this group is used to signal that session itself is completed and error variable contains error (or nil for no error). You need another group/channel for signaling stop

Reply:
👍

Reply:
fixd